packages/boxel-ui/addon/src/components/prerendered/index.gts

```diff
@@ -0,0 +1,44 @@
+import Component from '@glimmer/component';
+
+interface Signature {
+  Args: {
+    css?: string;
+    html?: string;
+  };
+  Blocks: {
+    default: [];
+  };
+}
+
+export default class Prerendered extends Component<Signature> {
+  constructor(owner: unknown, args: Signature['Args']) {
+    super(owner, args);
+
+    if (this.args.css) {
+      let styleElement = document.createElement('style');
```

Comment:
I just realized it would make sense to remove this element when component gets destroyed... Will add this